test/kwtool_web/controllers/auth_controller_test.exs

```diff
@@ -1,10 +1,31 @@
 defmodule KwtoolWeb.AuthControllerTest do
   use KwtoolWeb.ConnCase, async: true
 
-  describe "sign_up/2" do
+  describe "show/2" do
     test "returns 200 status", %{conn: conn} do
       conn = get(conn, "/sign_up")
-      assert html_response(conn, 200) =~ "Keyword Research Tool"
+
+      assert html_response(conn, 200) =~ "Registration"
+    end
+  end
+
+  describe "create/2" do
+    test "redirects to home page when the registration data is valid", %{conn: conn} do
+      password_inputs = %{password: "123456", password_confirmation: "123456"}
+      conn = post(conn, Routes.auth_path(conn, :create), user: params_for(:user, password_inputs))
+
+      assert redirected_to(conn) == Routes.page_path(conn, :index)
+      assert get_flash(conn, :info) == "Your account is created successfully!"
+    end
+
+    test "renders errors when registration data is invalid", %{conn: conn} do
```

Comment:
Let's use the word `params` instead of `data` to make the description more concise.
```suggestion
    test "renders errors when given invalid registration params", %{conn: conn} do
```

Reply:
updated.